app/src/androidTest/java/woowacourse/shopping/presentation/goods/list/GoodsActivityTest.kt

```
@@ -84,20 +136,117 @@ class GoodsActivityTest {
 
-        await.atMost(1, TimeUnit.SECONDS).until {
+        // then
+        await.atMost(2, TimeUnit.SECONDS).until {
             try {
-                onView(withId(R.id.btn_load_more))
-                    .perform(click())
+                onView(withId(R.id.btn_load_more)).perform(click())
+
+                onView(withId(R.id.btn_load_more)).check(matches(not(isDisplayed())))
+                true
+            } catch (e: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)